server/routerlicious/lerna-package-lock.json

```diff
@@ -506,7 +506,6 @@
 				"minimatch": "^3.0.4",
```

Comment:
The changes under `server/routerlicious` are unrelated to the code changes we're making. We should revert these.

Reply:
> The changes under `server/routerlicious` are unrelated to the code changes we're making. We should revert these.

Clean and rebuilt.